documentation/index.rst

```diff
@@ -0,0 +1,150 @@
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+Developer Interface
+===================
+
+Module
+------
+.. automodule:: mots.module
+    :members:
+
+
+Directory
+---------
+.. automodule:: mots.directory
+    :members:
+
+
+CLI
+---------
+.. automodule:: mots.cli
+    :members:
+
+
+Config
+---------
+.. automodule:: mots.config
+    :members:
+
+
+Utils
+---------
+.. automodule:: mots.utils
+    :members:
+
+Command Line Usage
+==================
+You can get this help by typing ``mots`` at the command line.
+
+usage: mots [-h] [--debug] {init,module,validate,clean,query} ...
+
+main command line interface for mots
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --debug               enable debug output
+
+commands:
+  {init,module,validate,clean,query}
+    init                initialize mots configuration in repo
```

Comment:
In this section, this is the output of `--help`, it's not generated. But I think it would be good to include this documentation somewhere in some format -- this is not the ideal format or way to include it though. I could reformat it and factor out some of the repetitive output to make it easier on the eyes.